backend/src/main/java/com/github/chuettenrauch/mixifyapi/security/config/SecurityConfig.java

```diff
@@ -19,6 +20,7 @@
 
 @Configuration
 @RequiredArgsConstructor
+@EnableMethodSecurity
```

Comment:
Ist auch notiert :D

Reply:
;) Das is nötig, um die `@PreAuthorize` Annotation nutzen zu können